docs/analytics_welcome/how_we_work.md

```diff
@@ -19,6 +19,7 @@ The section below outlines our team's primary meetings and their purposes, as we
 | -------- | -------- | -------- |
 | #**data-analysis** | Discussion | For sharing and collaborating on Cal-ITP data analyses. |
 | #**data-office-hours** | Discussion | A place to bring questions, issues, and observations for team discussion. |
+| #**data-warehouse-devs** | Discussion | For people building dbt models - focused on data warehouse performance considerations, etc. |
 | #**lunch-n-learn** | Discussion | A channel for discussions around our weekly Lunch n' Learns. |
```

Comment:
Could delete the lunch n learn channel mention since it is no longer relevant

Reply:
Resolved in [e6f2773](https://github.com/cal-itp/data-infra/pull/2602/commits/e6f2773037116a38d5e1b6783fa7c63729318178)